tests/neg/refutable-pattern-binding-messages.check

```diff
@@ -38,6 +22,22 @@
    | If the narrowing is intentional, this can be communicated by adding the `case` keyword before the full pattern,
    | which will result in a filtering for expression (using `withFilter`).
    | This patch can be rewritten automatically under -rewrite -source 3.2-migration.
+-- Error: tests/neg/refutable-pattern-binding-messages.scala:5:14 ------------------------------------------------------
+5 |  val Positive(p) = 5                     // error: refutable extractor
```

Comment:
Why did these error messages move? It seems non-optimal, as the messages are not in source order anymore.

Reply:
all that was changed was the warning became an error, so perhaps under `-Werror` true errors print last?